### DISCUSSION TITLE
LaserGRBL Configuration for Mecpow X3 Pro

### DISCUSSION THREAD

**Discussion by LDHTHOPI** [arkypita/LaserGRBL#2230] [Q&A]
Hi,

Does anyone have the configuration settings for the Mecpow X3 Pro 10W?

Thanks!

**Comment by Tribo-2dot0** [comment]
Página 23 do manual...
https://drive.google.com/file/d/1Drx5FfF0M_dabeh5k8tlxfl0siveOK7Q/view

**Reply by LDHTHOPI** [reply]
Thank you for your reply.  However, those instructions are for Lightburn software.  I am using LaserGRBL.

**Comment by LDHTHOPI** [comment]
It looks like LaserGRBL recognized my machine and created default settings.  I did need to make a slight adjustment to the X axis travel range and turn on $20 (soft limits).  I'll give it a try with the default settings and those two adjustments and see how it performs.
